include/mesh/nemesis_io.h

```diff
@@ -125,10 +125,18 @@ class Nemesis_IO : public MeshInput<MeshBase>,
 
 private:
 #if defined(LIBMESH_HAVE_EXODUS_API) && defined(LIBMESH_HAVE_NEMESIS_API)
-  Nemesis_IO_Helper * nemhelper;
-  int _timestep;
+  UniquePtr<Nemesis_IO_Helper> nemhelper;
 #endif
 
+  /**
+   * Keeps track of the current timestep index being written. Used
+   * when calling write_nodal_data() and other functions.
+   */
+  int _timestep;
+
```

Comment:
It seems inconsistent to have it inside, as none of the other members of the class are.  It is also consistent with the Exodus helper this way.